lib/test_track_rails_client/version.rb

```diff
@@ -1,3 +1,3 @@
 module TestTrackRailsClient
-  VERSION = "4.0.0.alpha35" # rubocop:disable Style/MutableConstant
+  VERSION = "4.0.0-rc1" # rubocop:disable Style/MutableConstant
```

Comment:
I think we weren't semver compliant by using a `.` instead of `-` 🤷

Reply:
i think there's something magic about the dot for gems - be careful that this doesn't cut a non-prerelease.

Reply:
oh interesting! i assumed that semver rules were the standard, i'll do a little more research.

Reply:
Rails does RC's with dots, for example: https://rubygems.org/gems/rails